src/routes/dashboard/components/Dashboard.js

```diff
@@ -38,21 +42,28 @@ export type PropsType = {|
 
 class Dashboard extends React.Component<PropsType> {
   getNavigationTileModels (cityCode: string, language: string): Array<TileModel> {
-    const { navigateToCategory, navigateToEvent, navigateToExtras, t, cities } = this.props
-
+    // check if news is enabled to show the menu item
```

Comment:
We always use a capital letter for first word in comments, please change accordingly (also other occurrences).